src/SFA.DAS.EmploymentCheck.AcceptanceTests/App.config

```diff
@@ -0,0 +1,63 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+  <configSections>
+    <section name="specFlow" type="TechTalk.SpecFlow.Configuration.ConfigurationSectionHandler, TechTalk.SpecFlow" />
+    <section name="nlog" type="NLog.Config.ConfigSectionHandler, NLog, Version=4.0.0.0, Culture=neutral, PublicKeyToken=5120e14c03d0593c" />
+  </configSections>
+  <specFlow>
+    <!-- For additional details on SpecFlow configuration options see http://go.specflow.org/doc-config -->
+  		<unitTestProvider name="NUnit" />
+  </specFlow>
+<startup>
+  <supportedRuntime version="v4.0" sku=".NETFramework,Version=v4.6.2" />
+</startup>
+  <appSettings>
+    <add key="ConfigurationStorageConnectionString" value="UseDevelopmentStorage=true;" />
+    <add key="EnvironmentName" value="LOCAL" />
+    <add key="ServiceName" value="SFA.DAS.EmploymentCheck" />
```

Comment:
Can we have a separate service or environment name for the tests?  I'd like to be able to keep local so I can run the app without switching config.

Reply:
In fact, it'll have to be a different environment name as all of the other configs will be different as well.